frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualization.tsx

```diff
@@ -23,12 +23,14 @@ const PipelineVisualization: React.FC<PipelineTopologyVisualizationProps> = ({
   }
 
   return (
-    <PipelineTopologyGraph
-      id={pipelineRun?.metadata?.name || pipeline.metadata.name}
-      nodes={nodes}
-      edges={edges}
-      layout={PipelineLayout.DAGRE_VIEWER}
-    />
+    <div style={{ marginBottom: 'var(--pf-global--spacer--md)' }}>
```

Comment:
why not marginBottom: global_spacer_md.value from patternfly/react-tokens? Both would do the same thing... I am unsure which is more appropriate.

Reply:
Was likely a hack to test something and stayed because it worked... probably should go into a scss file.